src/main/java/executor/command/CommandWeather.java

```diff
@@ -0,0 +1,147 @@
+package executor.command;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import executor.task.TaskList;
+import interpreter.Parser;
+import ui.Ui;
+import ui.Wallet;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class CommandWeather extends Command {
+    private LinkedHashMap<String, LinkedHashMap<String, String>> fullWeatherData;
+
+    /**
+     * CommandWeather displays weather information based on user request.
+     * @param userInput String is the user entered Input from CLI
+     */
+    public CommandWeather(String userInput) {
+        this.userInput = userInput;
+        this.commandType = CommandType.WEATHER;
+        this.description = "Command that displays weather for now, tomorrow or later";
+        setFullWeatherData(storeWeatherDataFromJson());
+    }
+
+    @Override
+    public void execute(TaskList taskList) {
+
+    }
+
+    @Override
```

Comment:
It is better to put the override function which has an output at the top after the constructor.

Reply:
Hey bro, I have updated that as requested!